packages/recovery-img/bootargs.cfg

```diff
@@ -0,0 +1,8 @@
+set kernel=/boot/vmlinuz
+if [ -n "$recoverylabel" ]; then
```

Comment:
this allow the image to be used both for booting in active/passive and for recovery

Reply:
giving another thought, we probably should extend this by default to all images otherwise those can't be used for recovery anymore. It also opens up to the possibilit of just use squashfs as well.

Reply:
that was done :+1: